terraform/cluster/aws/outputs.tf

```diff
@@ -34,7 +34,7 @@ output "route_table_public" {
 }
 
 output "subnets" {
-  value = aws_subnet.private.*.id
+  value = length(var.private_subnets_ids) == 0 ? aws_subnet.private.*.id : var.private_subnets_ids
```

Comment:
LGTM  but 
in case var.private=false the main resource `aws_eks_node_group.cluster` will use a public subnet. but it is not defined in the output.

Reply:
There's no need to define it on the output, since it is on the same module, the `aws_eks_node_group.cluster` can reference the resource directly, without using the output.

I don't know where this output is used, but originally it was referencing the private subnets, so I don't think there's a need to change it to use public subnets